tests/test_ingest.py

```diff
@@ -25,38 +25,71 @@
 import tempfile
 import unittest
 
-import lsst.daf.butler as dafButler
 import lsst.daf.butler.tests as butlerTests
+from lsst.daf.butler import DatasetType, Butler, DataCoordinate
+from lsst.daf.butler.core.utils import getFullTypeName
 
+from lsst.obs.base.ingest_tests import IngestTestBase
 from lsst.obs.base import RawIngestTask
 
 
 TESTDIR = os.path.dirname(__file__)
 
 
-class RawIngestTestCase(unittest.TestCase):
+class DummyCamRawIngestTask(RawIngestTask):
+    """For DummyCam we ingest a different dataset type that can return
+    a non-Exposure"""
+
+    def getDatasetType(self):
+        """Return the DatasetType of the datasets ingested by this Task.
+        """
+        return DatasetType("raw_dict", ("instrument", "detector", "exposure"), "StructuredDataDict",
+                           universe=self.butler.registry.dimensions)
+
+
+class RawIngestTestCase(IngestTestBase, unittest.TestCase):
```

Comment:
docstring? And note in the docstring that this TestCase doesn't test index files?